test/testUtil.ts

```diff
@@ -9,45 +9,180 @@ import path = require("path");
 import Q = require("q");
 
 export class TestUtil {
-
-    public static MOCK_SERVER_OPTION_NAME: string = "--mockserver";
-    public static PLATFORM_OPTION_NAME: string = "--platform";
-    public static TARGET_OPTION_NAME: string = "--target";
+    public static ANDROID_PLATFORM_OPTION_NAME: string = "--android";
+    public static ANDROID_SERVER_URL: string = "--androidserver";
+    public static ANDROID_EMULATOR: string = "--androidemu";
+    public static IOS_PLATFORM_OPTION_NAME: string = "--ios";
+    public static IOS_SERVER_URL: string = "--iosserver";
+    public static IOS_EMULATOR: string = "--iosemu";
+    public static RESTART_EMULATORS: string = "--clean";
     public static SHOULD_USE_WKWEBVIEW: string = "--use-wkwebview";
```

Comment:
I see now that this flag is a tristate ("true", "both", "false"). Should we have another "--use-uiwebview", so that if both are specified, it will run the tests for both, and if none are specified, run the tests for uiwebview by default?